docs/examples/segmentedcontrol/defaultExample.js

```diff
@@ -0,0 +1,27 @@
+// @flow strict
+import { type Node, useState } from 'react';
```

Comment:
This one's fine, as there are other non-type imports. So this will show up as
```js
import { useState } from 'react';
```
when the user views the code